### ISSUE TITLE
Having a `volume`   command

### ISSUE THREAD

**Issue by pedrolucasp** [docker/compose#11111]
### Description

Not much needs to be said I suppose. Having a `volume` command, context-aware of the services running in the compose file would make it a lot easier to delete, prune, and inspect volumes related to the services. The main usage for this is *not having to type the whole project name - service name - volume name, and not having to browse amidst all volumes you have available on a host.

If pointed in the right direction I can do my best to provide a patch for this.

**Comment by ndeloof** [comment]
Sounds reasonable to have such a command, you can look at the [list](https://github.com/docker/compose/blob/main/cmd/compose/list.go) command as an illustration of such "introspection" commands

**Comment by batcity** [comment]
@pedrolucasp (Just wondering) are you still planning on working on this feature?

**Comment by pedrolucasp** [comment]
> @pedrolucasp (Just wondering) are you still planning on working on this feature?

Yeah, I plan to, but got caught up with work stuff. Wen't as far as having the command available, but got stuck with connecting the backend part of it.

**Comment by batcity** [comment]
ah sounds good!, thanks for the update!
